src/App.js

```diff
@@ -44,53 +50,12 @@ function App() {
     <div className="App">
       <section>
         {/* if the player names are numbers, that will make our life easier later because we can reuse numbers as arrays. Note that this will make our app brittle! */}
-        <Player
-          to={to}
-          player={1}
-          hand={playerOneHand}
-          setFrom={setFrom}
-          selectedCard={selectedCard}
-          setTo={setTo}
-          setSelectedCard={setSelectedCard}
-        />
-        <Player
-          to={to}
-          player={2}
-          hand={playerTwoHand}
-          setFrom={setFrom}
-          selectedCard={selectedCard}
-          setTo={setTo}
-          setSelectedCard={setSelectedCard}
-        />
-        <Player
-          to={to}
-          player={3}
-          hand={playerThreeHand}
-          setFrom={setFrom}
-          selectedCard={selectedCard}
-          setTo={setTo}
-          setSelectedCard={setSelectedCard}
-        />
-        <CardList
-          cards={deck}
-          selectedCard={selectedCard}
-          setSelectedCard={setSelectedCard}
-          setFrom={setFrom}
-          player={'deck'}
-        />
-      </section>
-      <section>
-        {selectedCard && (
-          <ExecutePassButton
-            passCard={passCard}
-            setFrom={setFrom}
-            from={from}
-            to={to}
-            selectedCard={selectedCard}
-            setSelectedCard={setSelectedCard}
-          />
-        )}
+        <Player player={1} hand={playerOneHand} />
+        <Player player={2} hand={playerTwoHand} />
+        <Player player={3} hand={playerThreeHand} />
+        <CardList cards={deck} player={'deck'} />
```

Comment:
There is no need to pass in deck as a prop, it can be directly called in the CardList component.